backend/apis/parse-docket-pdf.api.js

```diff
@@ -25,6 +29,7 @@ app.post("/api/docket-pdfs", (req, res) => {
         .send({ error: "Uploaded file did not have mimetype application/pdf" });
     }
     fileWasUploaded = true;
+    if (filename) count++;
```

Comment:
busboy processes one file at a time. this is how I track the number of files were uploaded. I think messing with counters can be error prone. if anyone has a better solution, please share!

Reply:
no feedback. will resolve for now.